builds/checkin/dotnet.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
 ################################################################################
     displayName: Check pipeline preconditions (changes ARE NOT in either edgelet, docs, or mqtt folder)
     pool:
-      vmImage: "ubuntu-18.04"
+      vmImage: "ubuntu-20.04"
```

Comment:
does dotnet6 only run on ubuntu 20.04 or this was just an unrelated enhancement?

Reply:
Unrelated enhancement.